templates/profile.html

```diff
@@ -59,10 +59,6 @@ <h3 class="px-2 pt-0 pb-2 my-0 font-semibold">{{_('favourite_program')}}</h3>
                         </select>
                      </div>
                  {% endif %}
-                 <div class="flex flex-row gap-4 items-center border-gray-400 border p-4 rounded-lg">
-                    <input type="checkbox" name="agree_terms" id="agree_terms" required class="" {% if public_settings %}checked{% endif %}>
-                    <label for="agree_terms" class="cursor-pointer w-full">{{_('public_profile_info')}}</label>
-                 </div>
```

Comment:
Can you revert this change, please? This tick mark is used if a user wants their profile to be public, and we don't want to remove this one!

Reply:
The lines we want to remove are these ones: 

https://github.com/hedyorg/hedy/blob/f7823ac8ca3780dd680c3edef97a274a4f940afb/templates/profile.html#L145-L148

Can you please restore the lines you deleted and delete these ones instead? I can also do this if you want, but I don't have permissions to commit to your branch :c Thank you very much for your help @ghubrakesh!!

Reply:
See how we're looking to remove the ones that have the string `agree_third_party`. Wich are present in the lines I attached in 3 ways: the id of the input, the for in the label, and also in this weird string `{{_('agree_third_party')}}`, which is a label that will later be replaced with the appropriate translation in whichever language the user has set.

The one you removed only has the string: `agree_terms` which is for agreeing to have a public profile! It's perfectly reasonable to get confused, more so being your first time working with our codebase!

Reply:
Okay @jpelay, I will restore the changes made by me (I will add the deleted lines again) but what about the changes made by @Felienne? Should I ignore them and add the lines deleted just by me?

Reply:
Yes! The changes made by Felienne are correct, and you shouldn't revert them; just the ones I told you about.